test/functional/api/v2/auth_source_ldaps_controller_test.rb

```diff
@@ -53,4 +53,17 @@ class Api::V2::AuthSourceLdapsControllerTest < ActionController::TestCase
     show_response = ActiveSupport::JSON.decode(@response.body)
     assert !show_response[:success]
   end
+
+  test 'taxonomies can be set' do
+    AuthSourceLdap.any_instance.stubs(:test_connection).raises(Foreman::Exception)
+    put :update, { :id => auth_sources(:one).to_param,
+                   :organization_names => [taxonomies(:organization1).name],
+                   :location_ids => [taxonomies(:location1).id] }
+    show_response = ActiveSupport::JSON.decode(@response.body)
+    assert_response :success
+    assert_equal taxonomies(:location1).id,
```

Comment:
Notice this test tests both the PUT action accepts the right params, and the RABL in the response